opencensus/test_exporter.go

```diff
@@ -0,0 +1,98 @@
+package opencensus
```

Comment:
@rghetia what do you think about moving this somewhere in the OpenCensus ecosystem? It's the second time I implement it (the other time was internal, so I can't reuse it yet).

Reply:
I would put it as part of opencensus itself, something similar to what is done in open-telemetry [test exporter](https://github.com/open-telemetry/opentelemetry-go/blob/master/exporter/metric/test/test.go). Not sure if it is worth making that change in OC given that open-telemetry will be out in few months.

Reply:
I think it would be useful, I know people who are still developing with OpenCensus because OpenTelemetry isn't here now.

Let me know if you'd like me to send you a PR with this as a starting point, ultimately it's your decision as the maintainer.

Reply:
ok. Send me a PR.

Reply:
Will do.